### ISSUE TITLE
Split functionalities into seperate classes

### ISSUE THREAD

**Issue by miador** [MLH-Fellowship/prep-project-22.NOV.PREP.2#40]
Right now, all the functionality is overlapped in `App.js`, therefore it causes merge conflicts for all other PRs once one of them is merged into main. It would be a good practice to separate them in different classes in order to end this turmoil and the future features need to be implemented with this rule taken into account.

**Comment by miador** [comment]
Any volunteers for that?
